modules/post/windows/gather/screen_spy.rb

```diff
@@ -48,11 +50,21 @@ def record?
     datastore['RECORD']
   end
 
+  def process?
+    datastore['PROCESS']
+  end
```

Comment:
In Ruby process names that end with `?` should always return a boolean. I'd advise just naming this function `process` instead if it will not be returning a boolean.

```suggestion
  def process
    datastore['PROCESS']
  end
```

Reply:
I also question why this was added when it is never used in the code from what I can see.

Reply:
Fixed with https://github.com/rapid7/metasploit-framework/pull/14994/commits/0b06904dd020256ef43c039bf4d03dd01ef251cb